test/TEST-02-UNITTESTS/meson.build

```diff
@@ -3,15 +3,14 @@
 integration_tests += [
         integration_test_template + {
                 'name' : fs.name(meson.current_source_dir()),
-                'mkosi-args' : integration_test_template['mkosi-args'] + [
-                        '''--kernel-command-line-extra=
-
-frobnicate!
+                'cmdline' : integration_test_template['cmdline'] + [
```

Comment:
Why appended to the empty template? Why not to simply assign it?

Reply:
Same here, by appending to the empty template, we don't have to make large changes when we want to make the template non-empty later.